docs/notebooks/mister_p.ipynb

```diff
@@ -0,0 +1,8615 @@
+{
```

Comment:
<p>similar, how to read this chart and what's the conclusion</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/716/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
<p>Hope it's clearer now</p>